[2021-03-29 23:14:29,244] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_1 2021-03-28T00:00:00+00:00 [queued]>
[2021-03-29 23:14:29,294] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_1 2021-03-28T00:00:00+00:00 [queued]>
[2021-03-29 23:14:29,301] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-29 23:14:29,302] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-29 23:14:29,311] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-29 23:14:29,449] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_1> on 2021-03-28T00:00:00+00:00
[2021-03-29 23:14:29,460] {standard_task_runner.py:52} INFO - Started process 58 to run task
[2021-03-29 23:14:29,476] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_1', '2021-03-28T00:00:00+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp9idt3g9i', '--error-file', '/tmp/tmpf08eoguv']
[2021-03-29 23:14:29,487] {standard_task_runner.py:77} INFO - Job 7: Subtask runme_1
[2021-03-29 23:14:29,643] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_1 2021-03-28T00:00:00+00:00 [running]> on host 33dac7d1aa5e
[2021-03-29 23:14:29,824] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_1
AIRFLOW_CTX_EXECUTION_DATE=2021-03-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-28T00:00:00+00:00
[2021-03-29 23:14:29,828] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-29 23:14:29,834] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_1__20210328" && sleep 1
[2021-03-29 23:14:29,893] {bash.py:169} INFO - Output:
[2021-03-29 23:14:29,909] {bash.py:173} INFO - example_bash_operator__runme_1__20210328
[2021-03-29 23:14:30,921] {bash.py:177} INFO - Command exited with return code 0
[2021-03-29 23:14:30,970] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_1, execution_date=20210328T000000, start_date=20210329T231429, end_date=20210329T231430
[2021-03-29 23:14:31,042] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-29 23:14:31,080] {local_task_job.py:146} INFO - Task exited with return code 0
